examples/signature/factorizer_game.py

```diff
@@ -0,0 +1,86 @@
+# WARNING: this logic sig is for demo purposes only
```

Comment:
This example features in [my fork of the pyteal course](https://github.com/tzaffi/pyteal-course/blob/main/CONTRACT_PAYOUT.md) and will be useful to demonstrate graviton@🐗